core/src/main/resources/log4j4k8s.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<log4j2:Configuration status="WARN" name="${ff:instance.name}" strict="true" xmlns="http://logging.apache.org/log4j/2.0/config" xmlns:log4j2="log4j-config.xsd">
+	<!-- https://logging.apache.org/log4j/2.x/manual/customloglevels.html -->
+
+
+	<!--  This is for logging with Loki in kubernetes  -->
+	<Appenders>
+		<Appender name="json-stdout" type="Console">
+			<JsonTemplateLayout eventTemplateUri="classpath:EcsLayout.json"  />
+		</Appender>
+	</Appenders>
+
+	<Loggers>
+		<Root>
+			<AppenderRef ref="json-stdout"/>
```

Comment:
Did you verify the other log files are still used? I'm unsure if this overwrites it all or just appends to the log-configuration.